mne/io/ctf/markers.py

```diff
@@ -0,0 +1,68 @@
+# Author: Joan Massich <mailsik@gmail.com>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+import os.path as op
+from io import BytesIO
+
+from ...annotations import Annotations
+from .res4 import _read_res4
+
+
+def _get_markers(fname):
+    def consume(fid, predicate):  # just a consumer to move around conveniently
+        while(predicate(fid.readline())):
+            pass
```

Comment:
@larsoner do we have something similar somewhere? or do you find a way to use `dropwhile` from `itertools`?

Reply:
shall we make it an `io._utils`?